src/main/java/uk/yermak/audiobookconverter/fx/OutputController.java

```diff
@@ -105,6 +113,7 @@ private void initialize() {
                 Preset preset = Preset.instance(newValue);
                 ConverterApplication.getContext().setOutputParameters(preset);
             }
+            refreshSpeeds();
```

Comment:
Just wondering why do you need to refresh speed here... It should not be a problem, just trying to understand...

Reply:
Because the speed is not a part of the Preset, it resets when the preset changes.

Presets: https://github.com/yermak/AudioBookConverter/blob/development/src/main/java/uk/yermak/audiobookconverter/Preset.java#L19
Default speed:
https://github.com/yermak/AudioBookConverter/blob/development/src/main/java/uk/yermak/audiobookconverter/OutputParameters.java#L38

But the value in ComboBox still displays the previous one and needs to be reset. Maybe it should be moved to the `addOutputParametersChangeListener`, where other parameters updates.

By the way, it seems that the same issue reproduced with "Split by" ComboBox and we should update the display value.